sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -444,33 +444,86 @@ object CudfNthLikeAggregate {
  * In the future, this aggregate class should be removed and the mean values should be
```

Comment:
@ttnghia I don't know if this comment applies anymore. Do you see averages coming from libcudf's M2? If not, mind updating the comment (or removing it)?

Reply:
This is still relevant. I couldn't get libcudf to output the desired output to remove it.